docs/localization.md

```diff
@@ -136,3 +136,21 @@ namespace vcpkg::Commands
     }
 }
 ```
+## Selecting a Language
+
+Vcpkg chooses a language through the VSLANG environment variable, which should be set to a valid LCID (locale identifier, 4-byte value representing a language). Users can set VSLANG to any of the following LCIDs:
```

Comment:
```suggestion
vcpkg chooses a language through the `VSLANG` environment variable, which should be set to a valid LCID (locale identifier, 4-byte value representing a language). Users can set `VSLANG` to any of the following LCIDs:
```
For branding consistency we never capitalize vcpkg.

We should mention that if vcpkg is given an invalid LCID it falls back to English.